[2022-12-01T14:37:31.435+1300] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: load_kpi_view_dag.update_agg_table manual__2022-12-01T01:37:18.515867+00:00 [queued]>
[2022-12-01T14:37:31.448+1300] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: load_kpi_view_dag.update_agg_table manual__2022-12-01T01:37:18.515867+00:00 [queued]>
[2022-12-01T14:37:31.448+1300] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-12-01T14:37:31.448+1300] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-12-01T14:37:31.448+1300] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-12-01T14:37:31.459+1300] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): update_agg_table> on 2022-12-01 01:37:18.515867+00:00
[2022-12-01T14:37:31.466+1300] {standard_task_runner.py:55} INFO - Started process 69483 to run task
[2022-12-01T14:37:31.470+1300] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'load_kpi_view_dag', 'update_agg_table', 'manual__2022-12-01T01:37:18.515867+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/load_kpi_view.py', '--cfg-path', '/var/folders/xc/5xlsywvn2n5d0_ty56lp12y80000gp/T/tmp6wy_pgpl']
[2022-12-01T14:37:31.472+1300] {standard_task_runner.py:83} INFO - Job 225: Subtask update_agg_table
[2022-12-01T14:37:31.627+1300] {task_command.py:376} INFO - Running <TaskInstance: load_kpi_view_dag.update_agg_table manual__2022-12-01T01:37:18.515867+00:00 [running]> on host macbook-pro.hq.optimalbi.site
[2022-12-01T14:37:31.691+1300] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=load_kpi_view_dag
AIRFLOW_CTX_TASK_ID=update_agg_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-01T01:37:18.515867+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-01T01:37:18.515867+00:00
[2022-12-01T14:37:31.703+1300] {base.py:71} INFO - Using connection ID 'postgres_db' for task execution.
[2022-12-01T14:37:31.709+1300] {sql.py:315} INFO - Running statement: 
--Clear out the table
truncate logs.aggregation_table_new;
 --aggregation_view

insert into logs.aggregation_table_new
SELECT l.id,
        l.site,
        EXTRACT(year FROM l.start_timestamp) AS "Year",
        EXTRACT(month FROM l.start_timestamp) AS "Month",
        sum(EXTRACT(epoch FROM (l.end_timestamp - l.start_timestamp) / 60::double precision))::bigint AS outage_in_minutes
        FROM logs.all_sites l
        GROUP BY l.site, (EXTRACT(year FROM l.start_timestamp)), (EXTRACT(month FROM l.start_timestamp)), l.id
       , parameters: None
[2022-12-01T14:37:31.714+1300] {sql.py:324} INFO - Rows affected: 200
[2022-12-01T14:37:31.727+1300] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=load_kpi_view_dag, task_id=update_agg_table, execution_date=20221201T013718, start_date=20221201T013731, end_date=20221201T013731
[2022-12-01T14:37:31.793+1300] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-12-01T14:37:31.863+1300] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
